documentation/getting_started/README.md

```diff
@@ -0,0 +1,101 @@
+
+As a developer, you will likely be accessing a large range of internet/cloud services that require basic HTTP authentication or an API key. In most instances, such credentials are hard coded into applications and can be easily read with access to said applications. WoTT introduces credential management so that you can remotely download the necessary credentials onto your device as a JSON file. This removes the need for hard coded keys and logins from your application. 
+
+We do this via our Dasboard which you can use to monitor your devices. The Dashboard gives your device a trust score (based on how secure it is) as well as giving you the means to add and manage credentials for whatever purpose you need them for.
+
+## Installing the WoTT agent and claiming your device
+
+First, create an account for the [WoTT Dashboard](https://dash.wott.io) and login - you will need this to claim your device later.
+
+Currently, the Agent is only available to Debian package distributions such as Raspbian (for Pi) or Debian. You will also need to have `curl` installed.
+
+To install the agent, run the following in terminal:
+
+```
+$ curl -s https://packagecloud.io/install/repositories/wott/agent/script.deb.sh | sudo bash
+$ sudo apt install wott-agent
+```
+The terminal will now output your WoTT Device ID, claim token, and a link to claim the device. 
+Copy and paste this link into your browser- the fields may already have autofilled. Otherwise fill in the details in the relevant fields.
+
+![claim device](https://github.com/WoTTsecurity/wott-io/blob/master/documentation/getting_started/images/claim-device.png)
+
+**Note:** you can view your Device ID and claim token using the following commands as well:
+
+```
+$ wott-agent whoami
+$ wott-agent claim-token
+```
+Your device will now be registered and viewable on the WoTT Dashboard. You can now see the relative trust scores of your devices. To see device specific information, select it and add appropriate tags to these devices (this is relevant for credential management).
+
+See below for an example of a Raspberry Pi with its WoTT meta tags (automatically assigned based on hardware) and the user's personal tags for the Pi's purpose.
+
+![raspberry pi profile](https://github.com/WoTTsecurity/wott-io/blob/master/documentation/getting_started/images/rasbpi-profile.png)
+
+## Managing your own credentials through the WoTT Dashboard
+
+In scenarios where you might require API keys or logins for your applications, you can add and manage these credentials to the WoTT Dashboard. On the left of WoTT Dashboard is a sidebar.
+
+![sidebar](https://github.com/WoTTsecurity/wott-io/blob/master/documentation/getting_started/images/sidebar.png)
+
+Navigate to 'Credentials.' Select 'Add a credential'
+
+![credentials](https://github.com/WoTTsecurity/wott-io/blob/master/documentation/getting_started/images/credentials.png)
```

Comment:
So these need to be added in a slightly different fashion for the website. I realized I never told you that. You can find an example [here](https://raw.githubusercontent.com/WoTTsecurity/wott-io/master/_posts/2019-06-25-why-open-source-solutions-are-critical-for-IoT.md), but in short:

```
{% asset blog/iot-monster.png srcset:width="1300 2x" srcset:width="650 1x" alt="No Image" class="img-fluid" %}
```

(Where you need to tweak the variables.)